pinot-plugins/pinot-file-system/pinot-adls/src/main/java/org/apache/pinot/plugin/filesystem/ADLSGen2PinotFS.java

```diff
@@ -191,20 +182,22 @@ public void init(PinotConfiguration config) {
         clientSecretCredentialBuilder.httpClient(builder.build());
 
         dataLakeServiceClientBuilder.credential(clientSecretCredentialBuilder.build());
-        blobServiceClientBuilder.credential(clientSecretCredentialBuilder.build());
+        break;
+      }
+      case NONE: {
+        LOGGER.info("Authenticating using anonymous access");
         break;
       }
       default:
-        throw new IllegalStateException("Expecting valid authType. One of (ACCESS_KEY, AZURE_AD, AZURE_AD_WITH_PROXY");
+        throw new IllegalArgumentException(
```

Comment:
Should we try to initialize the client using `DefaultAzureCredential` to have a way to pass the credentials to azure client using ENV variables (or system properties)?

Reply:
Make sense, updated the default behavior to use `DefaultAzureCredentialBuilder`: https://learn.microsoft.com/en-us/java/api/com.azure.identity.defaultazurecredentialbuilder?view=azure-java-stable